boards/arm/ronoth_lodev/ronoth_lodev_defconfig

```diff
@@ -0,0 +1,33 @@
+# Ronoth LoDev board configuration
+# Copyright (c) 2021 Dean Weiten <dmw@weiten.com>
+# SPDX-License-Identifier: Apache-2.0
+
+# Zephyr Kernel Configuration
+CONFIG_SOC_SERIES_STM32L0X=y
+
+# Platform Configuration
+CONFIG_SOC_STM32L073XX=y
+
+# General Kernel Options
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=16000000
+
+# Enable MPU
+CONFIG_ARM_MPU=y
```

Comment:
@NorthernDean does the SoC implement the ARM MPU?

Reply:
Nevermind, i saw that it does; it is not defined in the SOC_SERIES, but on the SoC variant level. I would have expected it be added in the series (unless certain series variants do not implement it)